components/tab-switcher.jsx

```diff
@@ -0,0 +1,81 @@
+// Note: source code of this component is pretty much copied directly from https://github.com/mozilla/science.mozilla.org
```

Comment:
Why can't we use Mofo-UI's tab switcher? Because of the react version?  Routing works with this so I'm not really concerned right now given the timeline, just would like to move towards shared components as we can.